### PR TITLE
Condensed Viewer Setting

### LINKED ISSUE #734: Recordings should have a smaller viewer
Experiments show 450px is good.

### PR DESCRIPTION
**Pull request by tpendragon**
Closes #734

### CODE REVIEW

**Comment by hackmastera** [comment]
screenshot?

File: spec/views/catalog/_universal_viewer_default.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  context "when the exhibit is configured to have a condensed viewer" do
+    it "displays a condensed viewer" do
+      expect(rendered).to have_selector ".uv__overlay.condensed"
+    end
+  end
+
+  context "when the exhibit is configured not to have a condensed viewer" do
+    let(:exhibit) { FactoryBot.create(:exhibit, condensed_viewer: false) }
+    it "displays a condensed viewer" do
```
Comment (hackmastera) [inline spec/views/catalog/_universal_viewer_default.html.erb_spec.rb]: spec name needs to be updated

**Comment by tpendragon** [comment]
@hackmastera I did, we don't send model or change set information to Pomegranate, and there's no way for it to tell what kind of viewing experience it's about to generate even. This seemed like an easy compromise. If we start wanting to display a different (condensed vs not) experience for material, that seems like maybe something we can configure in Figgy down the line?

**Comment by tpendragon** [comment]
New:
Screen Shot 2020-02-20 at 4 14 53 PM

Old:
Screen Shot 2020-02-20 at 4 16 07 PM

**Pull request merged**